### DISCUSSION TITLE
Don't have sort function in list

### DISCUSSION THREAD

**Discussion by Georgefwt** [community/community#11439] [Other Feature Feedback, Questions, & Ideas] [answered]
In stars, we can sort repos like this:
<img width="682" alt="image" src="https://user-images.githubusercontent.com/49113320/153734910-5f1b5c18-df90-4846-9bdb-f46db4dc166d.png">
but we can't sort repos in the list.
I hope this function can be added.

**Comment by bailey** [comment, marked as answer]
Hey @Georgefwt! Thanks for this feedback, I agree that the convenience of filtering and sorting repositories is pretty vital when a list gets too long. I'll add your suggestion to our backlog so we can consider investing more time here. 👾
